src/functions.js

```diff
@@ -1,30 +1,49 @@
 function removeMiddle( words ){
   // words is an array which contains an odd number of strings
+  
   // return a new array containing only the middle word
   // the words array should no longer contain the middle word
   // hint: splice
+  return words.splice(Math.ceil(words.length / 2), 1);
 }
 
 function get2ndAnd3rd( myArray ){
   // myArray is an array of numbers
   // return an array containing the 2nd and 3rd items from myArray
   // myArray should remain unchanged
   // hint: slice
+  return myArray.slice(1, 3);
 }
 
 function mapper( myArray ){
   // myArray is an array of numbers
   // return a new array which has all items in myArray incremented by one
   // myArray should remain unchanged
   // hint: map
+  var newArray = myArray.map( function(arrItem) {
+    return arrItem + 1;
+  });
+  
+  return newArray;
+
 }
 
 function wordLengths( words ){
   // words is an array of strings
   // return a new array that contains the number of letters in each word
   // hint: strings have a 'length' property
+
+  var arr = [];
+
+  for (var i = 0; i < words.length; i++) {
```

Comment:
it's a good solution, but you could try to make it even easier using the `map` function 😉

Reply:
yes, it would be easier!  Thanks for the reviews!